.github/ISSUE_TEMPLATE/Documentation.yml

```diff
@@ -0,0 +1,79 @@
+name: Documentation issue
+description: >-
+  Report information that is missing, incomplete, vague, misleading or
+  plain wrong in online documentation such as the website, the README,
+  the wiki, etcetera.
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Thank you for taking the time to help us improve the
+        documentation.
+  - type: input
+    id: reference
+    attributes:
+      label: Reference to current documentation
+      description: >-
+        If you can identify an existing piece of documentation that is
+        lacking, please provide a URL (or multiple) below. If your
+        report is about missing information and there is no single
+        obvious place where it should be added, you can leave this
+        empty.
+      placeholder: "https://backbonejs.org/#Model-changedAttributes"
+  - type: textarea
+    id: quote
+    attributes:
+      label: Quote of current documentation
+      description: >-
+        If you provided a URL in the previous field, please quote the
+        problematic documentation below. This ensures that future
+        readers understand what you were responding to, in case the
+        referred page disappears or changes content.
+      placeholder: "
+        > Retrieve a hash of only the model's attributes that have
+        changed since the last
+        [set](https://backbonejs.org/#Model-set), or `false` if there
+        are none. Optionally, an external **attributes** hash can be
+        passed in, returning the attributes in that hash which differ
+        from the model. This can be used to figure out which portions
+        of a view should be updated, or what calls need to be made to
+        sync the changes to the server."
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem statement
+      description: >-
```

Comment:
I somewhat disagree here, it's asking for the effects rather than the cause. I know people generally find it more intuitive to discuss the cause before the effect, but it is the effect that actually motivates changes to the documentation. I find it important to start with the "why".

That said, your comment made me realize the naming of the field was a bit vague, so I renamed it to "effect".

Reply:
I agree with the name change